addons/Makefile.rel.am

```diff
@@ -0,0 +1,16 @@
+include %D%/src/Makefile.rel.am
+
+if ENABLE_ADDONS
+addonsdocdir = $(docdir)/addons
+dist_addonsdoc_DATA = \
+	%D%/README \
+	%D%/ChangeLog \
+	%D%/NEWS \
+	%D%/COPYING \
+	%D%/AUTHORS
```

Comment:
I don't really like how this got duplicated a lot… unfortunately I don't have a solution for that as we can't rely on something like `$(plugin)` anymore as it would conflict in the top-level Makefile